server/bin/pbench-index.py

```diff
@@ -330,6 +353,12 @@ def main(options, name):
                         es_res = es_index(
                             idxctx.es, actions, fp, idxctx.logger, idxctx._dbg
                         )
+                except SigIntException:
+                    idxctx.logger.exception("Continuing with next TarBall")
+                    continue
+                except SigTermException:
+                    idxctx.logger.exception("Breaking out of the loop")
```

Comment:
These should specifically call out the reason for the continue/break: for example,

`Indexing interrupted by SIGINT, continuing to next tarball`
`Indexing interrupted by SIGTERM, terminating`

And eventually, for `SIGHUP`, something like
`Indexing interrupted by SIGHUP, re-evaluating work`

Reply:
ok sure